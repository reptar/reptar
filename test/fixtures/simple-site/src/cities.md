---
title: Cities
slug: cities
date: 2016-02-11
---

Cities I want to go to:

{% for city in site.data.cities %}
  <p>Name: {{city.name}}</p>
  <p>Birthday: {{city.cuisine}}</p>
{% endfor %}
