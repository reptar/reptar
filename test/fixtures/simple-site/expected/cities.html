<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />

  <title>My Site Title</title>
  <meta name="description" content="Your website's description goes here.
">
  <meta name="keywords" content="" />

  <meta name="HandheldFriendly" content="True" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />

  <link rel="shortcut icon" href="/favicon.ico">

  <link rel="stylesheet" type="text/css" href="/css/main-d8c1b45cda.css" />
</head>
<body class="post-template">

  <header class="main-header">
    <div class="vertical">
      <div class="main-header-content">
        <h1 class="page-title">
          <a href="/">My Site Title</a>
        </h1>
      </div>
    </div>
  </header>

  <main id="content" class="content" role="main">
    

<article class="post">

  <header class="post-header">
    <h1 class="post-title">Cities</h1>
  </header>

  <section class="post-content">
    <p>Cities I want to go to:</p>
  <p>Name: New York City</p>
  <p>Birthday: bagel</p>
  <p>Name: Los Angeles</p>
  <p>Birthday: kale</p>
  <p>Name: New Orleans</p>
  <p>Birthday: beignets</p>

  </section>

</article>



    
  </main>

  <footer class="site-footer clearfix">
    <p class="footer-description">
      &copy; My Site Title
    </p>
    <p class="footer-published">
      joyfully published by <a href="https://github.com/reptar/reptar">Reptar</a>
    </p>
  </footer>

  <script type="text/javascript" src="/js/main.js"></script>
</body>
</html>
